docs/configuration/variables.rst

```diff
@@ -137,6 +137,11 @@ See :doc:`Companies</companies/companies_overview>` for more information.
    * - Zip Code (Company)
      - ``{contactfield=companyzipcode}``
 
+.. tip:: 
+  Custom Company Fields
+
+  The syntax for custom Company fields is different than the core Company field syntax. You must **not** add the word "Company" in the variable, but instead treat it like a "contactfield":
```

Comment:
Here it looks like you're going on to provide an example, but I can't see one. Is that intentional?

Also Vale picked up on the use of contactfield - do you mean Contact field?